exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -61,7 +64,7 @@
 public class ProfileResources {
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ProfileResources.class);
 
-  public final static int MAX_PROFILES = 100;
+  public final static int MAX_PROFILES = Integer.valueOf(System.getProperty(ExecConstants.HTTP_MAX_PROFILES, "100"));
```

Comment:
Make this part of `drill-override.conf`?

(Using an Inject, see [DrillRestServer](https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRestServer.java#L88))